<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QMainWindow" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>721</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>221</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="get_routeList">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Список маршрутов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getSchedule">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Расписание поездов</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="get_ticketReport">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Учет проданных билетов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reportButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Отчет</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="text">
      <string>IronTrans</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="schedule_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>731</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="schedule_table">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>621</width>
       <height>441</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Маршрут</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время отправления</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время прибытия</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QComboBox" name="schedule_filter">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>все</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Прошедшие</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Будущие</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="addTrain">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>510</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить рейс</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteTrain">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить рейс</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editTrain">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>510</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Редактировать рейс</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>От</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>До</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Искать</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateFrom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>yyyy-mm-dd</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateTo">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>yyyy-mm-dd</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="addTrain_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>521</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Введите дату рейса (в формате гггг-мм-дд)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputDate">
        <property name="styleSheet">
         <string notr="true">margin-left:150px;
font: 10pt &quot;Times New Roman&quot;;
margin-right:150px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Выберите название маршрута</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboRoute">
        <property name="styleSheet">
         <string notr="true">margin-left:150px;
font: 10pt &quot;Times New Roman&quot;;
margin-right:150px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addTrain_button">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top:10px;
margin-left: 200px;
margin-right:200px</string>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>519</width>
       <height>46</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Добавление нового рейса в расписание</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="backToSchedule">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="routes_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Список маршрутов</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backToBegin">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QListWidget" name="routeList">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>221</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="routes_table">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>381</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Город</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время прибытия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время отправления</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="deleteRoute">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>470</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить маршрут</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ticketReport_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Выберите дату:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="dateInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="cashierInfo">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>410</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Выберите кассира:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="dateList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="cashierList">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="back_fromReport">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>330</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Не продал(и) ни одного билета:</string>
     </property>
    </widget>
    <widget class="QLabel" name="bad">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>350</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Только кассиры на букву S</string>
     </property>
    </widget>
    <widget class="QLabel" name="best">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>290</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="editTrain_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1301</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>521</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Введите дату рейса (в формате гггг-мм-дд)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputDate_edit">
        <property name="styleSheet">
         <string notr="true">margin-left:150px;
font: 10pt &quot;Times New Roman&quot;;
margin-right:150px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Выберите название маршрута</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboRoute_edit">
        <property name="styleSheet">
         <string notr="true">margin-left:150px;
font: 10pt &quot;Times New Roman&quot;;
margin-right:150px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editTrain_button">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top:10px;
margin-left: 200px;
margin-right:200px</string>
        </property>
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>addTrain_widget</zorder>
   <zorder>editTrain_widget</zorder>
   <zorder>schedule_widget</zorder>
   <zorder>widget</zorder>
   <zorder>routes_widget</zorder>
   <zorder>ticketReport_widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Выйти из программы</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сменить пользователя</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
